src/constants.ts

```diff
@@ -4,3 +4,9 @@ export const SYSTEM_PROFILE_COLLECTION = 'system.profile';
 export const SYSTEM_USER_COLLECTION = 'system.users';
 export const SYSTEM_COMMAND_COLLECTION = '$cmd';
 export const SYSTEM_JS_COLLECTION = 'system.js';
```

Comment:
I wasn't sure of the best place to put this constant. I put it here, because we'll need it for both the source code and test files.

Reply:
Seems an expected place to put it.